src/kusto/azext_kusto/generated/_help.py

```diff
@@ -0,0 +1,558 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['kusto cluster'] = """
+    type: group
+    short-summary: kusto cluster
+"""
+
+helps['kusto cluster list'] = """
+    type: command
+    short-summary: Lists all Kusto clusters within a subscription.
+    examples:
+      - name: KustoClustersListByResourceGroup

```

Comment:
Yes, you can overwrite it in manual/_help.py. but I think that example name is defined in swagger?

Reply:
Yes, the names are driven from the example file name. I'll change it manually for this version and we will work on a better solution for the future auto-generated clients.

Reply:
@qiaozha You're right. The example name is defined in swagger. I think that's one major gap for us since cmdlet is different from swagger. Usually we cannot directly use the content of the swagger spec in the cmdlet. Another potential issue is the quality of the swagger spec. Even in the swagger spec, I would prefer having meaningful name of examples.